components/python/xattr/xattr-PYVER.p5m

```diff
@@ -0,0 +1,39 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Nona Hansel
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/xattr
```

Comment:
```
 newman  ~  touch xxx
 newman  ~  /usr/bin/xattr xxx 
Traceback (most recent call last):
  File "/usr/bin/xattr", line 11, in <module>
    load_entry_point('xattr==0.9.7', 'console_scripts', 'xattr')()
  File "/usr/lib/python3.5/vendor-packages/xattr/tool.py", line 196, in main
    attr_value = attr_value.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 36: invalid continuation byte
```

Reply:
I can't fix it and youtube-dl doesn't require it for standard usage.